frame/session/src/lib.rs

```diff
@@ -442,7 +442,11 @@ decl_storage! {
 			for (account, val, keys) in config.keys.iter().cloned() {
 				<Module<T>>::inner_set_keys(&val, keys)
 					.expect("genesis config must not contain duplicates; qed");
-				assert!(frame_system::Pallet::<T>::inc_consumers(&account).is_ok());
+				assert!(
+					frame_system::Pallet::<T>::inc_consumers(&account).is_ok(),
+					"Account ({:?}) does not exist at genesis to set key. Account not endowed?",
+					val,
```

Comment:
shouldn't it be account instead of val in parenthesis ?
```suggestion
					account,
```